straxen/plugins/bayes_peaklet_classification.py

```diff
@@ -0,0 +1,138 @@
+import numpy as np
+import numba
+from scipy.special import logsumexp
+import strax
+import straxen
+
+
+class BayesPeakletClassification(strax.Plugin):
+    """
+    Bayes Peaklet classification
+    Returns the ln probability of a each event belonging to the S1 and S2 class.
+    Uses conditional probabilities and data parameterization learned from wfsim data.
+
```

Comment:
Can you maybe add also the reference to a note here since this is something new.